CHANGELOG.md

```diff
@@ -41,6 +42,41 @@ title: Changelog
 - [0.7.0](#070)
 - [0.6.0](#060)
 
+## 2.6.0
+
+### Change
+
+- change(prometheus): redesign the latency metrics & update grafana [#3993](https://github.com/apache/apisix/pull/3993)
+- change(prometheus): don't expose metrics to internet [#3994](https://github.com/apache/apisix/pull/3994)
+- change(limit-count): ensure redis cluster name is set correctly [#3910](https://github.com/apache/apisix/pull/3910)
+- change: drop support of OpenResty 1.15 [#3960](https://github.com/apache/apisix/pull/3960)
+
+### Core
+
+- :sunrise: feat: support passing different host headers in multiple nodes [#4208](https://github.com/apache/apisix/pull/4208)
+- :sunrise: feat: add 50x html for error page [#4164](https://github.com/apache/apisix/pull/4164)
+- :sunrise: feat: support to use upstream_id in stream_route [#4121](https://github.com/apache/apisix/pull/4121)
+- :sunrise: feat: support client certificate verification [#4034](https://github.com/apache/apisix/pull/4034)
+- :sunrise: feat: add nacos support [#3820](https://github.com/apache/apisix/pull/3820)
+- :sunrise: feat: patch tcp.sock.connect to use our DNS resolver [#4114](https://github.com/apache/apisix/pull/4114)
+
+### Plugin
+
+- :sunrise: feat(redirect): support uri encoding [#4244](https://github.com/apache/apisix/pull/4244)
+- :sunrise: feat(key-auth): allow customizing header [#4013](https://github.com/apache/apisix/pull/4013)
+- :sunrise: feat(response-rewrite): allow using variable in the header [#4194](https://github.com/apache/apisix/pull/4194)
+- :sunrise: feat(ext-plugin): finish the first version [#4183](https://github.com/apache/apisix/pull/4183)
```

Comment:
add more comments: 

`With this feature, APISIX can support Java, Golang and other languages to implement custom plug-ins.`